android/build.gradle

```diff
@@ -45,6 +45,9 @@ repositories {
   google()
   jcenter()
   mavenCentral()
+  maven {
```

Comment:
This is covered by `google()` above. You should be able to remove this.

Reply:
Yup! That was added from the mocking libraries. Removed and it works as expected.